### PR TITLE
Made the list of CSS variables a table + Changes addon links to badges 

### PR DESCRIPTION
**Pull request by AndrewDaGuy**
Re-do of https://github.com/darkreader/darkreader/pull/5225 with addon links changed into badges

To-Do:
- Make the rules into a table :D

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -109,12 +109,14 @@ CSS
...
-
-- `--darkreader-neutral-background` should be mostly used for elements that have a wrong background color (usually bright backgrounds that should be dark).
-- `--darkreader-neutral-text` should be used for elements with a wrong text color (usually dark texts that should be light).
-- `--darkreader-selection-background` corresponds to user's Selection Background Color setting.
-- `--darkreader-selection-text` corresponds to user's Selection Text Color setting.
+Here is a full table of available CSS variables:
+
+| Variable                            	| Description                     	| Use                                                         	|
+|-------------------------------------	|---------------------------------	|-------------------------------------------------------------	|
+| `--darkreader-neutral-background`   	| Background Color                	| Mostly used for elements that have a wrong background color 	|
```
Comment (Gusted) [inline CONTRIBUTING.md:116]: I think the description could be somewhat longer. Something along the lines of `The neutral background color that correspondents to the user's settings`. The same with the text color.

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -109,12 +109,14 @@ CSS
...
+| `--darkreader-neutral-text`         	| Text Color                      	| Used for elements with a wrong text color                   	|
+| `--darkreader-selection-background` 	| User's Text Color setting       	| The user's Text Color setting                               	|
+| `--darkreader-selection-text`        	| User's Background Color setting 	| The user's Background Color setting                         	|
```
Comment (Gusted) [inline CONTRIBUTING.md:119]: Any idea why the indent for `|` is not like the other ones?

Comment (AndrewDaGuy) [reply CONTRIBUTING.md:119]: I think my formatter is having a seizure

**Comment by AndrewDaGuy** [comment]
Oh shit the commits added here

File: CONTRIBUTING.md (hunk 2 above)
Comment (AndrewDaGuy) [reply CONTRIBUTING.md:119]: I think this is an issue on your side 
image

File: CONTRIBUTING.md (hunk 1 above)
Comment (AndrewDaGuy) [reply CONTRIBUTING.md:116]: >  Something along the lines of `The neutral background color that correspondents to the user's settings.`


*Causally copy pastes*

File: CONTRIBUTING.md (hunk 2 above)
Comment (AndrewDaGuy) [reply CONTRIBUTING.md:119]: I will try to compact it more :]

File: CONTRIBUTING.md (hunk 1 above)
Comment (AndrewDaGuy) [reply CONTRIBUTING.md]: Done
image

File: CONTRIBUTING.md (hunk 2 above)
Comment (Gusted) [reply CONTRIBUTING.md]: LGTM

File: README.md [hunk 3]
```diff
@@ -1,8 +1,8 @@
...
-    
+<p align="center"><a rel="noreferrer noopener" href="https://chrome.google.com/webstore/detail/dark-reader/eimadpbcbfnmbkopoojfekhnkhdbieeh/"><img alt="Chrome Web Store" src="https://img.shields.io/badge/Chrome%20Web%20Store-141e24.svg?&style=flat-square&logo=google-chrome&logoColor=white"></a><a rel="noreferrer noopener" href="https://addons.mozilla.org/en-US/firefox/addon/darkreader/"><img alt="Firefox Add-ons" src="https://img.shields.io/badge/Firefox%20Addons-141e24.svg?&style=flat-square&logo=firefox-browser&logoColor=white"></a><a rel="noreferrer noopener" href="https://microsoftedge.microsoft.com/addons/detail/dark-reader/ifoakfbpdcdoeenechcleahebpibofpc/"><img alt="Edge Addons" src="https://img.shields.io/badge/Edge%20Addons-141e24.svg?&style=flat-square&logo=microsoft-edge&logoColor=white"></a><a el="noreferrer noopener" href="https://addons.thunderbird.net/en-US/thunderbird/addon/darkreader"><img alt="Thunderbird" src="https://img.shields.io/badge/Thunderbird-141e24.svg?&style=flat-square&logo=thunderbird&logoColor=white"></a>
```
Comment (Gusted) [inline README.md:4]:
```suggestion
<p align="center"><a rel="noreferrer noopener" href="https://chrome.google.com/webstore/detail/dark-reader/eimadpbcbfnmbkopoojfekhnkhdbieeh/"><img alt="Chrome Web Store" src="https://img.shields.io/badge/Chrome%20Web%20Store-141e24.svg?&style=flat-square&logo=google-chrome&logoColor=white&logoWidth=45"></a>  <a rel="noreferrer noopener" href="https://addons.mozilla.org/en-US/firefox/addon/darkreader/"><img alt="Firefox Add-ons" src="https://img.shields.io/badge/Firefox%20Addons-141e24.svg?&style=flat-square&logo=firefox-browser&logoColor=white&logoWidth=45"></a>  <a rel="noreferrer noopener" href="https://microsoftedge.microsoft.com/addons/detail/dark-reader/ifoakfbpdcdoeenechcleahebpibofpc/"><img alt="Edge Addons" src="https://img.shields.io/badge/Edge%20Addons-141e24.svg?&style=flat-square&logo=microsoft-edge&logoColor=white&logoWidth=45"></a>  <a el="noreferrer noopener" href="https://addons.thunderbird.net/en-US/thunderbird/addon/darkreader"><img alt="Thunderbird" src="https://img.shields.io/badge/Thunderbird-141e24.svg?&style=flat-square&logo=thunderbird&logoColor=white&logoWidth=45"></a>
```
Make it look a little bit bigger and add the 2 spaces between them to distance them.
image

**Review by Gusted — APPROVED** [review]
Thank you, definitely a good improvement to the new documentation looks.

**Pull request merged**